doc/project/testing.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: doc
+title: Testing
+---
+
+In this section we will discuss how to test Scala.js code managed as [Full cross project](./cross-build.html). All names for example subproject name such as `fooJVM` are taken from [cross compile example](https://github.com/scala-js/scalajs-cross-compile-example).
+
+
+## Directory Structure
+
+    <project root>
+     +- jvm
+     |   +- src/main/scala
+     |   +- src/test/scala
+     +- js
+     |   +- src/main/scala
+     |   +- src/test/scala
+     +- shared
+         +- src/main/scala
+         +- src/test/scala
+
+Similarly to `{shared|js|jvm}/src/main/scala` folders that contain your code, there can be also `{shared|js|jvm}/src/test/scala` folders that would contain your tests. Tests that verify correctness of shared code should go to `shared` test folder, while tests that check Scala JVM code or Scala.js code only should go to `jvm` or `js` test folders respectively. When you call `sbt> test` two iterations of tests will run:
```

Comment:
Try to cut long lines in the .md file so that they do not exceed 120 characters.

> When you call `sbt> test` two iterations of tests will run:

This is only true if there is a root project which `aggregate`s the JVM and JS parts. This should be explicit.